hoot-core/src/main/cpp/hoot/core/visitors/RemoveInvalidMultilineStringMembersVisitor.cpp

```diff
@@ -94,10 +94,20 @@ void RemoveInvalidMultilineStringMembersVisitor::visit(const ElementPtr &e)
                   rev->addElement(MetadataTags::RoleReviewee(), id2);
               }
             }
-            rev->removeElement(r->getElementId());
+            //  There is a one-to-one relation here, replace the multilinestring with the only way
+            if (rev->getMembers().size() == 1 && r->getMembers().size() == 1)
+              rev->addElement(MetadataTags::RoleReviewee(), r->getMembers()[0].getElementId());
+            //  Don't remove multilinestring relations that are members of a review relation
+            //  only contains one member that is the original multilinestring
```

Comment:
What this does is just move the single way from the `multilinestring` relation to the review relation and delete the `multilinestring`.  Before I was working under the impression that review relations contained at least two elements and that is why your example was crashing.  This fixes that assumption.